Makefile

```diff
@@ -406,7 +397,6 @@ integration-tests: all testrunner
 #     PLAYBOOK="<filename>" \
 #     RUNNER_ARGS="<...>" \
 #     TESTS="<...>" \  # Can be set to only run one or a few tests instead of the full test suite.
-#     DOCKER_VERSION=<...> \
```

Comment:
this shouldn't be removed - it refers to the docker version used to build VMs for integration testing